ci/release/changelogs/next.md

```diff
@@ -4,6 +4,7 @@
 
 - Opacity 0 shapes no longer have a label mask which made any segment of connections going through them lower opacity [#1940](https://github.com/terrastruct/d2/pull/1940)
 - Bidirectional connections are now animated in opposite directions rather than one direction [#1939](https://github.com/terrastruct/d2/pull/1939)
+- Animated connections in sketch mode are now generated in normal mode to fix imperfections in the path [#1944](https://github.com/terrastruct/d2/pull/1944)
```

Comment:
I think this doesn't need to be mentioned, it's just rolled up into the previous line you added.